app/src/main/convert.js

```diff
@@ -37,6 +37,23 @@ export default async function (exportOptions) {
     }
   });
 
+  ipcMain.once('cancel-processing', () => {
+    converter.cancel();
+    app.kap.mainWindow.send('export-progress', {text: ''});
```

Comment:
The above line is repeated 3 times in this file. Would be nice to dry it up.

Reply:
Actually, do we really need to call this here as it's already called on line 49?